types/pmx/tcr/lifecycle.schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$id": "https://akenza.io/pmx/tcr/lifecycle.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "lifecycle",
+  "title": "Lifecycle",
+  "properties": {
+    "batteryVoltage": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/device/schema.json#/$defs/batteryVoltage/volt"
+    }
+  },
+  "required": [
+    "batteryVoltage"
+  ]
+}
```

Comment:
Is there a reason to use two topics for lifecycle and id?
It looks to me as they would expect the same values.